packages/api-client-core/spec/Repeater.spec.ts

```diff
@@ -0,0 +1,1577 @@
+import {
+    Repeater,
+    RepeaterOverflowError,
+    DroppingBuffer,
+    FixedBuffer,
+    MAX_QUEUE_LENGTH,
+    SlidingBuffer,
+} from "@manufac/repeater";
+import { delayPromise } from "./helpers.js";
+
+describe("Repeater", () => {
```

Comment:
I think these tests should be in the forked package, not here, no? Either we should vendor both the code and the tests, or neither

Reply:
@airhorns Removed the tests completely. Once the PR is merged, I plan to migrate the forked Repeater package to this repo itself.